packages/samples/iife/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@coveo/iife-samples",
```

Comment:
Should we add something in scripts:deploy -> do nothing ?

So that we don't deploy any of those sample packages

Reply:
no need, it's already marked as `private`, and it doesn't contain a publish script.